website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md

```diff
@@ -228,9 +228,26 @@ See [Quitting Sessions]({{< ref "../drivers/#quitting-sessions" >}}).
 
 ## Running Selenium File
 
-{{< alert-code >}}
-Add example for how to execute each of the above pages via command line
-{{< /alert-code >}}
+{{< tabpane text=true >}}
+{{< tab header="Java" >}}
+{{< badge-code >}}
+{{< /tab >}}
+{{< tab header="Python" >}}
+{{< badge-code >}}
+{{< /tab >}}
+{{< tab header="CSharp" >}}
+{{< badge-code >}}
+{{< /tab >}}
+{{< tab header="Ruby" >}}
+{{< gh-codeblock path="examples/ruby/spec/getting_started/first_script.rb#L23" >}}
```

Comment:
We don't want the instructions to be comments in the code. It could be a link directing to the README file.